client/src/reducers/colors.js

```diff
@@ -66,6 +66,7 @@ const ColorsReducer = (
       };
     }
 
+    case "color by gene set":
```

Comment:
is this implemented?  Is color-by-gene-set any different than color-by-continuous?

Reply:
The UI is there, but the gene `aggregate` functionality is still outstanding and won't be resolved in this PR.

Reply:
Unless I'm mistaken, triggering this action will cause an error.  I would remove this until it is implemented.

Reply:
Removed.